### PR TITLE
Custom workout programs, videos, photos

### PR DESCRIPTION
**Pull request by dongyuzheng**

### CODE REVIEW

**Review by gautamgupta — APPROVED** [review]
Code-wise looks good, didn't review business logic.

File: backend/api/v1/endpoints/media.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        )
+
+
+class PhotoSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Photo
+        fields = (
+            'id',
+            'exercise',
```
Comment (pobeng) [inline backend/api/v1/endpoints/media.py]: do we need the exercise field when capturing photos and videos? I was originally thinking just a basic representation with no reference to exercise or nutrition

File: backend/api/v1/endpoints/workout_day.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+class WorkoutDaySerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = WorkoutDay
+        fields = (
+            'exercise',
+            'day_of_week',
+            'sets',
```
Comment (pobeng) [inline backend/api/v1/endpoints/workout_day.py:13]: what's this field for again? Thought it was an implicit field

Comment (dongyuzheng) [reply backend/api/v1/endpoints/workout_day.py:13]: its implicit dw

Comment (dongyuzheng) [reply backend/api/v1/endpoints/workout_day.py:13]: just for display.

**Pull request merged**